### ISSUE TITLE
Can't add users on touch screen device 

### ISSUE THREAD

**Issue by jriberg** [emqx/emqx#12886]
#### Environment

- OS: Android and Windows 
- Erlang/OTP: 
- EMQ: 0.5.0(latest?)

#### Description

*A description of the issue*
Installed i my Home Assistant, added auth backend internal DB and then when pressing users the whole row gets dragged instead

**Comment by keynslug** [comment]
> EMQ: 0.5.0(latest?)

Hi! This doesn't look like a version string. It would help to know the exact EMQX version you're experiencing this with.

**Comment by jriberg** [comment]
Sorry. Home assistant add-on version was 0.5.0, EMQX is v5.6.0.
Have tried it both in home assistant and stand alone with docker-compose

**Comment by keynslug** [comment]
Thanks! Turns out the dashboard is touchscreen-aware, but unfortunately does not currently works well enough on small screen devices. This is a known issue. We recommend using medium or large screen devices in the meantime.

**Comment by jriberg** [comment]
I tried it on my Samsung S7+ tablet, wouldn't that be considered somewhat
big screen at least?

Anyhow, good that it's known so the issue can be dealt with :)

**Comment by keynslug** [comment]
CC @ysfscream.

**Comment by ysfscream** [comment]
@jriberg Yes, first of all, thank you for your feedback. We should indeed optimize for smaller device screens. However, I just tested it on my iPad and found it works.

![IMG_0171](https://github.com/emqx/emqx/assets/21299158/8266228e-b9a6-4f75-a089-090f3dbc6020)

so for now, we still recommend using a PC or other devices with higher screen resolutions. Thank you.

**Comment by zmstone** [comment]
closing this issue as there is no activity for over 2 years.
